js/twister_newmsgs.js

```diff
@@ -207,7 +225,18 @@ function getNewDMsCount() {
     var newDMs = 0;
 
     for (var user in _newDMsPerUser) {
-        if (_newDMsPerUser[user])
+        if (!user[0] === '*' && _newDMsPerUser[user])
```

Comment:
@slr same... this doesn't work here, i think it should be `if (user[0] !== '*')`